assets/src/edit-story/components/library/panes/media/media3p/media3pCategories.js

```diff
@@ -33,19 +34,32 @@ const CategorySection = styled.div`
   flex-wrap: wrap;
 `;
 
-// TODO(#2362) Wire up pill list to state and remove these fake pills.
-const fakePillList = ['COVID-19', 'Nature', 'Wallpapers', 'People'];
-const selectedPill = 'Nature';
-
-const Media3pCategories = () => (
+const Media3pCategories = ({
+  categories,
+  selectedCategory,
+  selectCategory,
+  deselectCategory,
+}) => (
   <CategorySection>
-    {
-      // TODO(#2362) Wire up pill list to state and remove these fake pills.
-      fakePillList.map((e, i) => (
-        <CategoryPill isSelected={e == selectedPill} key={i} title={e} />
-      ))
-    }
+    {categories.map((e) => {
+      const selected = e.name === selectedCategory;
```

Comment:
Is there some sort of category id in the state we could use instead of the name here? 

I doubt that the name will be repeated, but it might be better to use an ID to check if selected rather than a name in case the API does give us a conflict for some reason.

Reply:
As discussed offline, the `name` is the ID. Just our naming is a bit weird.